spec/helpers/regions_helper_spec.rb

```diff
@@ -56,7 +56,8 @@
 
     it 'renders the snippets when present' do
       rendered_region = region_content(@region)
-      rendered_region.should have_selector('h1', content: 'Fred Flinstone')
+      rendered_region.should have_selector('h1', text: 'Fred Flinstone')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.